### PR TITLE
Upgrade ERNIE-M Tokenizer

### PR DESCRIPTION
**Pull request by joey12300**
### PR types

Bug fixes
### PR changes

Others
### Description

Upgrade ERNIE-M tokenizer and the relative faster tokenizer

### CODE REVIEW

File: faster_tokenizer/faster_tokenizer/include/models/unigram.h [hunk 1]
```diff
@@ -40,6 +41,8 @@ struct Unigram : public Model {
   virtual std::vector<std::string> Save(
       const std::string& folder,
       const std::string& filename_prefix) const override;
+  void SetFilterToken(const std::string& filter_token);
```
Comment (ZeyuChen) [inline faster_tokenizer/faster_tokenizer/include/models/unigram.h]: 补充注释

**Review by ZeyuChen — COMMENTED** [review]
关键环节和函数补充注释

File: faster_tokenizer/faster_tokenizer/include/models/unigram.h (hunk 1 above)
Comment (joey12300) [reply faster_tokenizer/faster_tokenizer/include/models/unigram.h]: Done

**Review by ZeyuChen — APPROVED** [review]
LGTM

**Pull request merged**